packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -182,6 +182,7 @@ const COLUMNS = [
 
 const GROUPED_ACTIONS = [
   {
+    disabled: vdis => some(vdis, ['type', 'VDI-unmanaged']),
```

Comment:
`some(vdis, { type: 'VDI-unmanaged' })`

(homogeneity, readability and maintainability)